blocks/plain-text/style.native.scss

```diff
@@ -0,0 +1,9 @@
+.blocks-plain-text {
```

Comment:
Did you check if it isn't included in web build by accident?

Reply:
Not extensively, I didn't. What I only tested was modifying values in the style file and see if GB-web picks those up, visually. Tried out adding a `background-color` for example. Any practical way to test it out more thoroughly perhaps?

Reply:
I can test it myself before merging, but I think if you put `background-color: pink` in the RN version, it shouldn't get applied in the web version. That's all we need to ensure :)

Reply:
That's grand, that was the test I did and it succeeded (color wasn't picked up) so, feel free to test is your side as well, thanks!